client/Lykke.Service.ConfirmationCodes.Client/IConfirmationCodesClient.cs

```diff
@@ -61,6 +61,24 @@ public interface IConfirmationCodesClient
         [Delete("/api/SmsConfirmation")]
         Task DeleteSmsCodeAsync([Body] SmsConfirmationRequest model);
 
+        /// <summary>
+        /// Requests Google 2FA setup for the client
+        /// </summary>
+        [Post("/api/Google2FA/RequestSetup")]
+        Task<RequestSetupGoogle2FaResponse> Google2FaRequestSetupAsync([Body] RequestSetupGoogle2FaRequest model);
```

Comment:
I think this can be done in a REST manner:
[post, put, get]
/api/Google2FA/Setup

Reply:
it can, but PUT doesn't fit